src/WebJobs.Script.Scaling/AzureWorkerTable.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public sealed class AzureWorkerTable : IWorkerTable
+    {
+        public const string AzureWorkerTableName = "azureworkertable";
```

Comment:
You should consider applying the Azure -> AppService renames to the table name and these types, if you agree